Jakefile.js

```diff
@@ -1031,3 +1159,85 @@ task("lint-server", ["build-rules"], function() {
         lintWatchFile(lintTargets[i]);
     }
 });
+
+function ProgressBar(title) {

```

Comment:
Shouldn't this be in its own file? And I thought we already used an existing progress bar. Why write our own?